frontend/javascripts/admin/tasktype/task_type_create_view.tsx

```diff
@@ -48,7 +48,7 @@ type FormValues = {
     mergerMode: boolean;
     preferredMode?: APIAllowedMode;
     allowedModes: APIAllowedMode[];
-    resolutionRestrictions: APIResolutionRestrictions;
+    magRestrictions: APIMagRestrictions;
```

Comment:
Maybe this renaming here is the cause for the task type mag restrictions not working on this branch. Maybe the backend still expects the old name `resolutionRestrictions` ? @fm3?

Reply:
hmm, no, resolutionRestrictions should no longer exist anywhere :thinking:  We’ll have a look.

Reply:
the form item still had old names :(

Reply:
Adapting the names seems to fix it. Editing the task type now shows the restricted mags. Zooming out in a task with the task type with restricted mags yields the error "Annotating data is restricted to a certain zoom range. (...)".